torch/testing/_internal/common_methods_invocations.py

```diff
@@ -240,6 +241,19 @@ def _getattr_qual(obj, name, default=_NOTHING):
         else:
             raise
 
+
+# test if a tensor is close to an integer
```

Comment:
More precisely this creates a boolean mask for the tensor that is True where the tensor is "close" to an integer and False where it isn't